app/survey_config/social_survey_config.py

```diff
@@ -58,3 +60,26 @@ def get_footer_legal_links(self, cookie_has_theme: bool) -> Optional[list[dict]]
             ]
 
         return None
+
+
+@dataclass
+class WelshSocialSurveyConfig(
+    SocialSurveyConfig,
+):
+    def __post_init__(self):
+        super().__post_init__()
+
+        base_url: str = ACCOUNT_SERVICE_BASE_URL_SOCIAL
+
+        language_code = "cy"
+
+        self.account_service_log_out_url: str = f"{self.base_url}/{language_code}/start"
+
+        self.contact_us_url: str = f"{CY_ONS_URL}/aboutus/contactus/surveyenquiries"
+        self.cookie_settings_url: str = f"{base_url}/{language_code}/cookies/"
```

Comment:
Slightly confusing that there are both `base_url` and `self.base_url` being used interchangeably here - does `base_url` need renaming?

Reply:
That declaration was unnecessary in child class, removed now, so only `self.base` is used when needed. Thanks!